semver.md

```diff
@@ -102,8 +104,10 @@ intended compatibility requirements as denoted by its associated
 normal version. Examples: 1.0.0-alpha, 1.0.0-alpha.1, 1.0.0-0.3.7,
 1.0.0-x.7.z.92.
 
-1. Build metadata MAY be denoted by appending a plus sign and a series of dot
-separated identifiers immediately following the patch or pre-release version.
+1. Build metadata MAY be denoted by appending a plus sign followed by one
+or more identifiers. The plus immediately follows the patch or
```

Comment:
Should also say "plus sign" like in the line above.

Reply:
Indeed, pushed a fix.